packages/paypal-commerce-integration/src/validation-schemas/getPaypalCommerceRatePayValidationSchema.ts

```diff
@@ -49,6 +49,6 @@ export default memoize(function getPaypalCommerceRatePayValidationSchema({
 
             return schema;
             // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
-        }, {} as { [key: string]: StringSchema }),
+        }, {} as { [key: string]: StringSchema<string | null> }),
```

Comment:
Is it possible to not use type assertion here?

Reply:
@bc-dronov we need to use it here because of https://github.com/bigcommerce/checkout-js/pull/1599/files#diff-1560885ed55ae002f8c50c5d683bd78a8427a4ed1864413210aa8f8efa662271R23